lambdas/test/utils/health.spec.ts

```diff
@@ -0,0 +1,130 @@
+import { HealthStatus, refreshContentServerStatus } from '@katalyst/lambdas/apis/status/health'
```

Comment:
We should probably also move the test in a similar fashion to the helper

Reply:
moved the file